modules/video/src/tracking/tracker_vit.cpp

```diff
@@ -90,21 +90,26 @@ static void crop_image(const Mat& src, Mat& dst, Rect box, int factor)
     Rect roi(x1 + x1_pad, y1 + y1_pad, x2 - x2_pad - x1 - x1_pad, y2 - y2_pad - y1 - y1_pad);
     Mat im_crop = src(roi);
     copyMakeBorder(im_crop, dst, y1_pad, y2_pad, x1_pad, x2_pad, BORDER_CONSTANT);
+
+    return crop_sz;
 }
 
 void TrackerVitImpl::preprocess(const Mat& src, Mat& dst, Size size)
 {
-    Mat mean = Mat(size, CV_32FC3, params.meanvalue);
-    Mat std = Mat(size, CV_32FC3, params.stdvalue);
-    mean = dnn::blobFromImage(mean, 1.0, Size(), Scalar(), false);
-    std = dnn::blobFromImage(std, 1.0, Size(), Scalar(), false);
-
     Mat img;
     resize(src, img, size);
 
     dst = dnn::blobFromImage(img, 1.0, Size(), Scalar(), false);
     dst /= 255;
-    dst = (dst - mean) / std;
+    int inner_size = size.width * size.height;
+    for (int i = 0; i < 3; i++) {
+        float _mean = params.meanvalue[i],
+              _std = params.stdvalue[i];
+        auto *dst_ptr = dst.ptr<float>() + i * inner_size;
+        for (int j = 0; j < inner_size; j++) {
+            dst_ptr[j] = (dst_ptr[j] - _mean) / _std;
+        }
+    }
```

Comment:
why not just `blobFromImageWithParams`?

Reply:
Good catch! I was not familiar with this API. Let me have a try with this.